configure.ac

```diff
@@ -1092,6 +1092,11 @@ if test "x$enable_sysusers" != "xno"; then
 fi
 AM_CONDITIONAL(ENABLE_SYSUSERS, [test "$have_sysusers" = "yes"])
 
+AC_ARG_ENABLE(gshadow, AS_HELP_STRING([--disable-gshadow], [disable shadow group support]))
+AS_IF([test "x${enable_gshadow}" != "xno"], [
+        AC_DEFINE(HAVE_GSHADOW, 1, [shadow group support is enabled])
+])
```

Comment:
also, please name this "ENABLE_GSHADOW" rather than "HAVE_GSHADOW"... we generally use HAVE_xyz to indicate the existance of some dep, while ENABLE_xyz indicates whether some specific logic is enabled...